netutils/webclient/webclient.c

```diff
@@ -182,15 +182,21 @@ struct conn_s
 #define	WGET_FLAG_GOT_CONTENT_LENGTH 1U
 #define	WGET_FLAG_CHUNKED            2U
 
+struct wget_target_s
+  {
+    char scheme[sizeof("https") + 1];
+    char hostname[CONFIG_WEBCLIENT_MAXHOSTNAME];
+    char filename[CONFIG_WEBCLIENT_MAXFILENAME];
+    uint16_t port;     /* The port number to use in the connection */
+  };
```

Comment:
```suggestion
{
  char scheme[sizeof("https") + 1];
  char hostname[CONFIG_WEBCLIENT_MAXHOSTNAME];
  char filename[CONFIG_WEBCLIENT_MAXFILENAME];
  uint16_t port;     /* The port number to use in the connection */
};
```
BTW, do you know why `scheme` is `sizeof("https") + 1` and not `sizeof("https")`?

Reply:
seems like +1 is not needed. I will add checking this to my TODO list :)